gym_multigrid/envs/fourrooms.py

```diff
@@ -50,9 +50,8 @@ class FourRooms(MultiGridEnv):
 
     def __init__(
         self,
-        agent_pos=None,
-        goal_pos=None,
-        grid_size: tuple = (19, 19),
+        env_seed: int = 0,
```

Comment:
The env seed has to be in the reset() method based on the standard gymnasium conventions